networking/network_security/configuring-ipsec-ovn.adoc

```diff
@@ -8,16 +8,16 @@ toc::[]
 
 With IPsec enabled, you can encrypt both internal pod-to-pod cluster traffic between nodes and external traffic between pods and IPsec endpoints external to your cluster. All pod-to-pod network traffic between nodes on the OVN-Kubernetes cluster network is encrypted with IPsec in _Transport mode_.
 
-IPsec is disabled by default. It can be enabled either during or after installing the cluster. For information about cluster installation, see xref:../../installing/overview/index.adoc#ocp-installation-overview[{product-title} installation overview].
-
 [IMPORTANT]
 ====
-If your cluster uses link:https://www.redhat.com/en/topics/containers/what-are-hosted-control-planes[{hcp}] for Red Hat {product-title}, IPsec is not supported for IPsec encryption of either pod-to-pod or traffic to external hosts.
+If you enabled the ESP hardware offload feature for a bond interface, do not enable IPsec. Enabling IPsec for this configured interface can cause installation or upgrade issues for your cluster. 
```

Comment:
I am not sure this perspective makes a lot of sense because I don't think we have much more use cases of ESP hardware offload other than IPsec and this is documentation about "how to enable IPsec" so is understood that the user wants to enable IPsec.

I would rather say something like
```
The use of ESP hardware offload when enabling IPsec is not supported and can cause network issues for your cluster.
```

Also since ESP hardware offload could be enabled by default, shouldn't we document how to disable it?

And I thought this problem was specific to vsphere but here we seem to be implying this is a general problem?

And I have yet another question: since which version has this been a problem? Why haven't we seen this before 4.15?

@pperiyasamy you might be the best person to answer some of this questions.